eksctl/01_setup.md

```diff
@@ -0,0 +1,99 @@
+# eksctl setup
```

Comment:
TODOs
- [ ] Apply every change of the FE and API
- [ ] Write how to deploy ajktown application step by step (of what you did)

Reply:
handled.